test/PlaidClientTest.js

```diff
@@ -937,6 +973,84 @@ describe('plaid.Client', () => {
         ], cb);
       });
     });
+    describe('deposit switch', () => {
+      const getAccessToken = (cb) => {
+        pCl.importItem(
+          ['identity', 'auth'],
+          {'user_id': 'user_good', 'auth_token': 'pass_good'},
+          (err, response) => {
+            expect(err).to.be(null);
+            expect(response).to.be.ok();
+            expect(response.access_token).to.be.ok();
+            cb(null, response.access_token);
+          });
+      };
+
+      const getAccountId = (access_token, cb) => {
+        pCl.getAccounts(access_token, (err, response) => {
+          expect(err).to.be(null);
+          expect(response).to.be.ok();
+          expect(response.accounts).to.be.ok();
+          cb(null,
+            {
+               account_id: response.accounts[0].account_id,
```

Comment:
could you perhaps filter for account type/subtype instead of using the 0th index?

Reply:
⚡️